### PR TITLE
Java branch

### PR DESCRIPTION
**Pull request by elenka-sh**
First homework completed

### CODE REVIEW

File: Java/Task1/src/Main.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+   public class Main {
+     //main class declaration
```
Comment (Hmeliumm) [inline Java/Task1/src/Main.java:4]: Please move it above the commented line with class declaration or remove it

File: Java/Task1/src/Main.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+     // System.out.println("My first program");*/
```
Comment (Hmeliumm) [inline Java/Task1/src/Main.java:5]: Please remove it

File: Java/Task1/src/Main.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    public  static void  main(String[] args) {
+
+    Scanner sc = new Scanner(System.in);
+    System.out.println("Hello. What is your name?");
+    String name = sc.nextLine();
+    System.out.println("Where are you live  " + name + "?");
+    String adress = sc.nextLine();
+    System.out.println((adress));
```
Comment (Hmeliumm) [inline Java/Task1/src/Main.java:13]: Please remove extra brackets "()"

File: Java/Task1/src/PersonInformation.java [hunk 4]
```diff
@@ -0,0 +1,2 @@
+public class PersonInformation {
```
Comment (Hmeliumm) [inline Java/Task1/src/PersonInformation.java:1]: Please move you code in Main class here and delete Main class

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: Java/Task1/src/PersonInformation.java (hunk 4 above)
Comment (elenka-sh) [reply Java/Task1/src/PersonInformation.java]: Could you see chenges please.

File: Java/Task1/src/PersonInformation.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import java.util.Scanner;
+
+    //PersonInformation class declaration
+    public class PersonInformation {
```
Comment (Hmeliumm) [inline Java/Task1/src/PersonInformation.java:4]: Please format code: Ctrl+Alt+L

File: Java/Task1/src/PersonInformation.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    public  static void  main(String[] args) {
+
+    Scanner sc = new Scanner(System.in);
+    System.out.println("Hello. What is your name?");
+    String name = sc.nextLine();
+    System.out.println("Where are you live  " + name + "?");
+    String adress = sc.nextLine();
+    System.out.println(adress);
+    System.out.println( name + "; " + adress);
```
Comment (Hmeliumm) [inline Java/Task1/src/PersonInformation.java:13]: Minor: Please change ; to ,

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**